.eslintrc

```diff
@@ -625,6 +625,22 @@
       "globals": {
         "tinymce": "readonly"
       }
+    },
+    {
+      "files": [
+        ".storybook/main.cjs"
+      ],
+      "rules": {
+        "node/no-extraneous-require": "off"
+      }
+    },
+    {
+      "files": [
+        ".storybook/stories/playground/story-editor/api/fonts.js"
+      ],
+      "rules": {
+        "import/no-unresolved": "off"
```

Comment:
Same here, I'd just add an `eslint-disable-next-line` comment in that specific file.

That said, I definitely need to fix this fonts import thing.

Reply:
I'll take a look too 😄